CHANGES.rst

```diff
@@ -1,7 +1,18 @@
 Changes
 =======
 
-Version 0.5.0 (2018-04-23)
+Version 0.6.0 (2019-12-20)
+--------------------------
+
+- Fixes bug with big file uploads by using data streaming.
+- Adds user login endpoints using OAuth, currently configured to work with CERN
+  SSO but extensible to use other OAuth providers such as GitHub, more in `Invenio-OAuthClient <https://invenio-oauthclient.readthedocs.io/en/latest/>`_.
+- Adds endpoints to integrate with GitLab (for retrieving user projects and creating webhooks). GitLab instance to connect to configurable through environment variable ``REANA_GITLAB_HOST``.
+- Adds new endpoint ``/me`` to retrieve user information.
+- Improves security by allowing only requests with ``REANA_URL`` in the host header, avoiding host header injection attacks.
```

Comment:
``Improves security by allowing requests only with ``REANA_URL`` in the host header, avoiding host header injection attacks.``